### ISSUE TITLE
left/right leg parameters not consistent

### ISSUE THREAD

**Issue by osrf-migration** [osrf/drcsim#295]
**Original report ([archived issue](https://osrf-migration.github.io/drcsim-gh-pages/#!/osrf/drcsim/issues/295)) by DavidC (Bitbucket: [DavidC](https://bitbucket.org/%7Bfcd58e93-ca10-45f7-b2db-c913c5878cc3%7D/)).**

----------------------------------------
Looking at the bounds calculation output (2.6.1) , I noticed inconsistency with l_leg_uhz and r_leg_uhz.

```
#!c++
[ INFO] [1369159631.928631237]: Bounds for joint[l_leg_uhz] is [0.100000, 458.333333], model default is [1.000000]
[ INFO] [1369159631.928651037]: Bounds for joint[l_leg_mhx] is [0.100000, 750.000000], model default is [1.000000]
[ INFO] [1369159631.928669755]: Bounds for joint[l_leg_lhy] is [0.100000, 1083.333333], model default is [1.000000]
[ INFO] [1369159631.928692759]: Bounds for joint[l_leg_kny] is [0.100000, 916.666667], model default is [1.000000]
[ INFO] [1369159631.928711580]: Bounds for joint[l_leg_uay] is [0.100000, 916.666667], model default is [1.000000]
[ INFO] [1369159631.928729196]: Bounds for joint[l_leg_lax] is [0.100000, 375.000000], model default is [1.000000]
[ INFO] [1369159631.928758077]: Bounds for joint[r_leg_uhz] is [0.100000, 1083.333333], model default is [1.000000]
[ INFO] [1369159631.928793890]: Bounds for joint[r_leg_mhx] is [0.100000, 750.000000], model default is [1.000000]
[ INFO] [1369159631.928830808]: Bounds for joint[r_leg_lhy] is [0.100000, 1083.333333], model default is [1.000000]
[ INFO] [1369159631.928864529]: Bounds for joint[r_leg_kny] is [0.100000, 916.666667], model default is [1.000000]
[ INFO] [1369159631.928899175]: Bounds for joint[r_leg_uay] is [0.100000, 916.666667], model default is [1.000000]
[ INFO] [1369159631.928935709]: Bounds for joint[r_leg_lax] is [0.100000, 375.000000], model default is [1.000000]


```

**Comment by osrf-migration** [comment]
**Original comment by Steve Peters (Bitbucket: [Steven Peters](https://bitbucket.org/%7B2ccfed09-18b8-4921-8d58-15ef01092802%7D/), GitHub: [scpeters](https://github.com/scpeters)).**

----------------------------------------
This had been fixed in [pull request #191](https://osrf-migration.github.io/drcsim-gh-pages/#!/osrf/drcsim/pull-requests/191), but it must have gotten lost in a merge.

**Comment by osrf-migration** [comment]
**Original comment by Steve Peters (Bitbucket: [Steven Peters](https://bitbucket.org/%7B2ccfed09-18b8-4921-8d58-15ef01092802%7D/), GitHub: [scpeters](https://github.com/scpeters)).**

----------------------------------------

* changed state from "new" to "resolved"

[pull request #318](https://osrf-migration.github.io/drcsim-gh-pages/#!/osrf/drcsim/pull-requests/318), thanks for catching this
